tests/resources/telescope_positions-North-ground.ecsv

```diff
@@ -34,7 +34,7 @@
 #       - {CLASS: null, DESCRIPTION: null, ID: null, SITE: null, SUBTYPE: null, TYPE: null}
 #       DOCUMENT:
 #       - {CREATION_TIME: null, ID: null, TITLE: null, TYPE: null, URL: null}
-#     INSTRUMENT: {CLASS: null, DESCRIPTION: null, ID: null, SITE: CTA-North, SUBTYPE: null, TYPE: null}
```

Comment:
This made me realize that we still have a lot of references to "CTA" in the code, whereas it should now be "CTAO". Should we open an issue to change that?

Reply:
I wouldn't spend time on it now but only when a new version of the metadata documents is released. There might be other changes to be done.

Reply:
Ok, I just created an issue such that we don;t forget about it. No need to work on that now